### ISSUE TITLE
What is the power connector

### ISSUE THREAD

**Issue by wes-b** [microsoft/Azure-Kinect-Sensor-SDK#745]
I want to make my own power connector. What is the size and type of connector used to power the Azure Kinect?

**Comment by wes-b** [comment]
The connector is a 4.5mm OD with 3.0mm ID and a pin diameter of 0.6mm. Similar to this connector: https://www.alibaba.com/product-detail/4506-DC-plug-4-5mm-0_60742810909.html?spm=a2700.7724857.normalList.20.61e761fcUIgZxg

**Comment by greatwallet** [comment]
Is there any recommended amperage?

**Comment by lukemc** [comment]
Amperage is 1.5 min and in my experience they really mean minimum.

I did get those Alibaba cores out of desperation and made a few by casting sugru around the barrel but it wasn't ideal. The minimum order was a bit silly so if anyone is stuck and needs some find me on twitter (lukemcneice) and I'm sure I can ship you some.

I did find another product (from a 'HP Notebook') that I could butcher and ended up just splicing off the useful part and remaking the cable using these  instead: https://www.amazon.co.uk/gp/product/B08K3K6LKR/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1
